app/src/pages/inside/stepPage/stepGrid/stepGrid.jsx

```diff
@@ -319,6 +319,7 @@ export class StepGrid extends Component {
           events: {
             onEditEvent:
               events.MAKE_DECISION_MODAL_EVENTS && events.MAKE_DECISION_MODAL_EVENTS.openModal,
+            getOnClickIssueTicket: events.getOnClickIssueTicket,
```

Comment:
This line will be unnecessary in case of you add `getOnClickIssueTicket` inside `getMakeDecisionModalEvents` in `modalEventsCreators` file.